lib/graphql/schema.rb

```diff
@@ -221,6 +221,22 @@ def types
       end
     end
 
+    # Returns a list of Arguments and Fields referencing a certain type
+    # @param type_name [String]
+    # @return [Hash]
+    def get_members_of_type(type_name)
```

Comment:
What do you think of calling this `#get_references_to` (or `#references_to`)? Interfaces and Unions have "members", but I don't think of `User.handle` as a member of `String`, how does that sound to you?